jenkins-integration/beaker/install/shared/30_customize_settings.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+
+test_name 'Configure settings on SUT'
+
+def get_puppet_settings_from_env()
+  ENV['PUPPET_GATLING_PUPPET_SETTINGS']
+end
+
+puppet_settings = JSON.parse(get_puppet_settings_from_env())
+
+Beaker::Log.notify("INITIALIZING PUPPET SETTINGS to: #{puppet_settings}")
+
+sections = puppet_settings.keys
+sections.each do |s|
+  Beaker::Log.notify("Updating puppet settings for section '#{s}'")
+  puppet_settings[s].each do |setting, value|
+    Beaker::Log.notify("In section '#{s}', setting '#{setting}' to '#{value}'")
+    on(master, puppet("config set #{setting} #{value} --section #{s}"))
+  end
+end
+
+puppet_conf = on(master, "cat /etc/puppetlabs/puppet/puppet.conf").stdout.chomp
```

Comment:
Fair enough.  Can ignore it for now and look at it again when we might want to do some p3 testing.  Wasn't sure if you had any more plans for PE 3.3 testing in the works or what not.